test/Chainweb/Test/Rosetta/RestAPI.hs

```diff
@@ -389,7 +389,7 @@ genesisId = BlockId 0 "rdfJIktp_WL0oMr8Wr6lH49YkERAJ9MlFp0RPLMXPDE"
 rosettaVersion :: RosettaNodeVersion
 rosettaVersion = RosettaNodeVersion
     { _version_rosettaVersion = "1.3.1"
-    , _version_nodeVersion = "2.1"
+    , _version_nodeVersion = "2.1.1"
```

Comment:
```suggestion
    , _version_nodeVersion = VERSION_chainweb
```
Requires CPP, but makes this self-updating

Reply:
👍 on that change @larskuhtz